docs/victory-voronoi-container/docs.md

```diff
@@ -0,0 +1,132 @@
+# VictoryVoronoiContainer
+
+`VictoryVoronoiContainer` calculates a voronoi diagram based on the data of every child component it
```

Comment:
_API comment_

As a user, I'm a little confused by why/how this is a "container". It seems like an implementation detail that I don't need to worry about. In fact, that might apply to all of the container components. 

Rather than containers, these feel more like "mix-ins" or "add-ons"